package.json

```diff
@@ -19,29 +19,33 @@
     "npm": ">=5.6.0"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free": "5.3.1",
     "bootstrap-sass": "3.3.7"
   },
   "devDependencies": {
     "@frctl/fractal": "1.1.7",
     "access-sniff": "3.2.0",
     "cheerio": "1.0.0-rc.2",
+    "css": "2.2.4",
     "del": "3.0.0",
     "fs-extra": "5.0.0",
     "gulp": "4.0.0",
     "gulp-accessibility": "3.1.1",
+    "gulp-cheerio": "0.6.3",
     "gulp-file": "0.4.0",
     "gulp-if": "2.0.2",
-    "gulp-inline-svg": "1.2.0",
+    "gulp-pretty-data": "0.1.2",
     "gulp-rename": "1.2.2",
     "gulp-sass": "3.2.1",
     "gulp-sass-lint": "1.3.4",
     "gulp-svgmin": "1.2.4",
+    "gulp-svgstore": "7.0.1",
     "gulp-tap": "1.0.1",
     "gulp-trim": "1.0.2",
     "hiff": "0.3.0",
+    "indent": "0.0.2",
+    "list-stream": "1.0.1",
     "node-sass": "4.9.3",
     "pretty": "2.0.0",
-    "slug": "^0.9.2"
+    "svg4everybody": "2.1.9"
```

Comment:
of, zou dit eigenlijk zelf een van de peerDependencies moeten worden... Of stellen we dat het de verantwoordelijkheid van de developers is om ervoor te zorgen dat er 'iets' wordt ingevlogen als fallback voor slechte/oude browsers?

Reply:
Dit is een `devDependency` omdat de toolkit dit nodig heeft om een goed-werkende build te produceren.

Zie #414 en de memo uit #414. De conclusie is dat developers zelf iets gaan maken waarmee ze IE issues afvangen. De developers zijn vrij te kiezen wat ze willen, daarom geen `peerDependency` of `dependency`

Reply:
Ok. Helder.